client/src/pages/transaction.tsx

```diff
@@ -44,15 +52,22 @@ export function Transaction() {
         ) : (
           "Unknown"
         ),
+        Request: <Code maxW="50em">{data.request}</Code>,
+        Response: <Code maxW="50em">{data.response}</Code>,
       }
     : {};
 
   if (data?.method === "ledger.send") {
+    const token = knownTokens.find((t) => t.address === data.argument.symbol);
     txn = {
       ...txn,
       From: <Code>{data.argument.from}</Code>,
       To: <Code>{data.argument.to}</Code>,
-      Amount: <Text>{data.argument.amount.toLocaleString()} MFX</Text>,
+      Amount: token
+        ? `${(data.argument.amount / token.precision).toLocaleString()} ${
+            token.ticker
+          }`
```

Comment:
Divide the amount by the precision and get the ticker name from the placeholder list.